examples/custom-colors/README.md

```diff
@@ -7,11 +7,101 @@ Javascript example
 - [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/custom-colors/index.html)
 - to run locally, see the [explanations in the repository README](../../README.md#running-examples-locally)
 
+
+## ♻️ BPMN Visualization Usage
 Override the BPMN element styles using various ways. `mxGraph` knowledge is required to handle style changes.
 See https://process-analytics.github.io/bpmn-visualization-js/#bpmn-support-howto-elements-rendering for more information.
 
-Content
-- override default fill and stroke colors: update the `StyleConfigurator` method prototypes
+:warning: In order to avoid to have to many content in the README, we simplify it. You can find all the content of the example in [custom-colors.js](custom-colors.js).
+
+
+Content:
```

Comment:
I agree that explaining what is done in the code can help people to understand how it works.
But there is too much code here and this will become a maintenance issue: we would be forced to update the README while changing the code in the js file.
Why not being more general here and add comments in the js code?

Reply:
Personnally, with all the code in the JS file, I found it difficult to know what was really interesting. Maybe we need to cut the file.
But, when I read a tutorial, I prefer to have a snippet of code which explain each features, and after a repo/file/... with all the code together 😄

Reply:
I understand this and I agree but the examples are currently provided to demonstrate what the lib can do but not in the way we want to support.
This is more a demonstrator of the lib capabilities.

Here we are documenting something we don't want to keep, and IMHO it would be more efficient to work on the lib extensibility first then update the examples. When this is done, we can provide all information about the code in its final shape and supported way.